src/JunctionTree.jl

```diff
@@ -1633,7 +1633,7 @@ end
 
 Return the variable order stored in a tree object.
 """
-getVariableOrder(treel::AbstractBayesTree)::Vector{Symbol} = treel.variableOrder
+DistributedFactorGraphs.getVariableOrder(treel::AbstractBayesTree)::Vector{Symbol} = treel.variableOrder
```

Comment:
tree not defined in DFG, is it?  Or is this a function name override issue?

If so, we should rather import the name from DFG and override here for dispatch.  I'm a little confused on why calling DFG explicitly here is the right thing to do.